packages/react-topology/src/pipelines/components/nodes/TaskNode.tsx

```diff
@@ -45,8 +45,8 @@ export type TaskNodeProps = {
   badgeTextColor?: string;
   badgeBorderColor?: string;
   badgeClassName?: string;
-  badgePopoverProps?: string;
```

Comment:
Removing props is considered a breaking change. Rather than removing it, maybe just log a warning that this prop is not used for anything.

Reply:
It was never used for anything. If anyone did set it (only one consumer thus far and they are not) then it was pointless.
I'd rather remove it and avoid any confusion that would be caused by auto-complete coming up with this prop.

Reply:
This prop was never used. Anyone using it (only one adopter thus far and they are not) would not have gotten any use from doing so.

I would rather remove it to avoid confusion that would be cause by auto-complete showing this parameter.

Reply:
WDYT @tlabaj

Reply:
@jeff-phillips-18 can you at least bump the major version if remove the prop?